src/cli/dev.test.ts

```diff
@@ -147,7 +147,7 @@ describe("dev command", () => {
       throw new Error("exit");
     });
 
-    await expect(program.parseAsync(["node", "test", "dev", "--yes"])).rejects.toThrow("exit");
+    await expect(program.parseAsync(["npx", "pepr", "dev", "--yes"])).rejects.toThrow("exit");
```

Comment:
```suggestion
    await expect(program.parseAsync(["dev", "--yes"], { from: "user" })).rejects.toThrow("exit");
```

Consistency with other tests we've written, though it's functionally equivalent